modules/file-integrity-understanding-file-integrity-cr.adoc

```diff
@@ -3,19 +3,15 @@
 // * security/file_integrity_operator/file-integrity-operator-understanding.adoc
 
 [id="understanding-file-integrity-custom-resource_{context}"]
-=  Understanding the `FileIntegrity` custom resource
+=  Understanding the File Integrity custom resource
```

Comment:
Any reason to update this line? `FileIntegrity` is more reasonable. It is a customer resource name.